.vsts-ci.yml

```diff
@@ -256,8 +256,12 @@ stages:
           - Darwin
         publishUsingPipelines: true
         pool:
-          name: NetCorePublic-Pool
-          queue: buildpool.windows.10.amd64.vs2017.open
+          ${{ if eq(variables['System.TeamProject'], 'public') }}:
+            name: NetCorePublic-Pool
+            queue: buildpool.windows.10.amd64.vs2017.open
+          ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+            name: NetCoreInternal-Pool
+            queue: buildpool.windows.10.amd64.vs2017
```

Comment:
It looks like line 250 already ensures that the TeamProject is not public, so I don't think we need the separate conditions here.

Reply:
This pool doesn't need to be a helix build pool. It doesnt' download anything except a few asset manifest artifacts.